lib/google_drive/session.rb

```diff
@@ -418,6 +423,24 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
+    # Returns GoogleDrive::Collection with given +id+.
```

Comment:
I'm confused about why this is showing up again.  Seems like it should already be in master via the merge of #313.